raml/sfdc_api.raml

```diff
@@ -0,0 +1,52 @@
+# RAML 0.0
+title:
+version: v1
+baseUrl: http://example.com/api
+/alert:
+  post:
+    description: Create or update Case in SFDC
+    responses:
+      200:
+    body:
+      application/json:
+        schema |
+          {"$schema": “http://json-schema.org/schema”,
+            "type":"object"
+            "description": "Alert Data"
+            "properties": {
+              "description":   {"type": "json"},
+              "alertId":       {"type": "string"},
+              "subject":       {"type": "string"},
+              "environment":   {"type": "string"},
+              "alertPriority": {"type": "string"},
+              "alertHost":     {"type": "string"},
+              "alertService":  {"type": "string"}
+          example: |
+            alert: {
+            {
+              "description":   {
+                "affected_hosts": [
```

Comment:
why is this in description?
I would prefer to see this outside of description in separated root key